geometry_msgs/Pose car_pose
geometry_msgs/Pose block_pose
float64[] xs
float64[] ys
float64[] thetas
